benchmarks/args.hpp

```diff
@@ -32,7 +36,9 @@ struct args
         if (argc != 4)
         {
             is_valid = false;
+#ifndef PURE_MPI
```

Comment:
just to clarify: we don't have any need to print configs here when pure MPI is used? I can't think of anything either right now, so seems reasonable to me.

Reply:
@boeschf One thing would be whether we use `wait` or `waitall`, or `test` vs `testany`, but I think we can skip that. I'd rather have those tests independent, in case sb. wants to use them.